src/mithril.postgrest.js

```diff
@@ -37,14 +37,16 @@
     var loader = m.prop(defaultState), d = m.deferred();
     loader.load = function(){
       loader(true);
+      m.redraw();
+      m.startComputation();
```

Comment:
It does not make much sense tu use this with ```background: true``` in the request.
Maybe it's better to remove the start/end computation and the ```background: true```